ocaml/xapi/xapi_pbd.ml

```diff
@@ -124,9 +124,9 @@ let resync_cluster_stack_for_sr_type ~__context ~sr_sm_type =
      multiple clusters, we may want to change this behaviour. *)
   required_clusters
   |> List.iter
-    (fun (cluster_ref, cluster_rec) ->
+    (fun (_,_) ->
        Helpers.call_api_functions ~__context (fun rpc session_id ->
-           Client.Client.Cluster.pool_resync ~rpc ~session_id ~self:cluster_ref)
+           Client.Client.Cluster.pool_resync ~rpc ~session_id)
     )
```

Comment:
I could move the cluster_stack matching logic to `host_resync`?

Reply:
Its fine here, cluster stack is more SM related, see my other comment on simplifying this code by using find_cluster_host.